.github/workflows/docs.yml

```
@@ -0,0 +1,32 @@
+name: github pages
+permissions:
+  contents: write
+on:
+  push:
```

Let's trigger the deploy only on docs changes :
```suggestion
  push:
    paths:
      - "docs/**"
```